_stylesheet.scss

```diff
@@ -270,6 +272,7 @@ $dock_style_modes: [null, shrink, extended, extended-shrink];
 
 .dash-item-container .overview-tile .app-grid-running-dot {
     margin-bottom: $dash_padding_gnome - 4px;
+    //offset-y: -$dash_padding + $dash_edge_offset;
```

Comment:
It doesn't seem to work in this form. But removing the `//` appears to work.

Reply:
Also `dash_edge_offset` isn't reused so probably doesn't need to be a variable.

Reply:
Agh! I was doing some tests with and without to ensure that it worked fine, and clearly forgot to remove the comment. Where's the emoji for "banging the head against a wall"?

Reply:
Fixed.